### PR TITLE
[Minor] Improve missing name error message in Gravitino CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Explain to the user what is missing and how to fix it.

### Why are the changes needed?

See above.

Fix: # N/A

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Locally.

### CODE REVIEW

File: clients/cli/src/main/java/org/apache/gravitino/cli/ErrorMessages.java [hunk 1]
```diff
@@ -29,7 +29,7 @@ public class ErrorMessages {
   public static final String UNKNOWN_SCHEMA = "Unknown schema name.";
   public static final String UNKNOWN_TABLE = "Unknown table name.";
   public static final String MALFORMED_NAME = "Malformed entity name.";
-  public static final String MISSING_NAME = "Missing name.";
+  public static final String MISSING_NAME = "Missing --name option.";
```
Comment (shaofengshi) [inline clients/cli/src/main/java/org/apache/gravitino/cli/ErrorMessages.java:32]: Will "--" be redundant if saying it is an "option"?

Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/ErrorMessages.java:32]: Perhaps, don't really care either way, was just making it explicit for the user,

Comment (shaofengshi) [reply clients/cli/src/main/java/org/apache/gravitino/cli/ErrorMessages.java:32]: Ok, get it.

**Review by shaofengshi — APPROVED** [review]
LGTM

**Pull request merged**